server/routes.go

```diff
@@ -20,6 +21,7 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/chewxy/math32"
```

Comment:
Instead of an outside dependency use `math.Sqrt`

Reply:
math.Sqrt only accepts float64, so pretty sure we have to use this or revert normalization to cast sum as a float64

Reply:
Let's try casting